server.js

```diff
@@ -97,14 +97,15 @@ server.register({ register: require('./api') }, function(err) {
     return;
   }
 
-  var endpoint = process.env.PUBLIC_PROJECT_ENDPOINT;
-  var port = endpoint.match(/:(\d+)/);
+  const endpoint = process.env.PUBLIC_PROJECT_ENDPOINT;
+  let port = endpoint.match(/:(\d+)/);
+
   if (!port) {
     return;
   }
 
-  var port = parseInt(port[1]);
-  require('mox-server').runServer(port, function() {
-    console.log('running mox server on port', port);
+  port = parseInt(port[1], 10);
```

Comment:
parseInt actually changed in ES6, it has been redefined to parse base 10 unless otherwise specified, unlike ES5, where this was not the case. As such, you don't need the `parseInt( ..., 10)` change here.